AssessmentService/src/main/java/com/revature/caliber/intercoms/CategoryClient.java

```diff
@@ -0,0 +1,16 @@
+package com.revature.caliber.intercoms;
+
+import org.springframework.cloud.netflix.feign.FeignClient;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import com.revature.caliber.beans.Category;
+
+@FeignClient(name="category", url="localhost:9090/")
```

Comment:
We would like to configure the url from conf files instead of the url property in code as it will need to change when we deploy